features/device-store/org.wso2.carbon.device.mgt.enterprise.store.feature/src/main/resources/publisher/samples/assets/deviceTypes/install.js

```diff
@@ -0,0 +1,164 @@
+/*
+ Description: The following class is used to the devices provided in this directory.
+ Filename: install.js
+ Created Date: 9/11/15.
+ */
+var installer = function () {
+
+	var log = new Log();
+	var DESIRED_LIFECYCLE_STATE = 'Published';
+	var DEFAULT_LIFECYCLE = 'DeviceLifeCycle';
+	var START_STATE='Initial';
+	var robot=require('/modules/automation/lifecycle.robot.js').robot();
+
+	var lifeCycleRobot=new robot.LifeCycleRobot();
+	lifeCycleRobot.init({lifecycle:DEFAULT_LIFECYCLE});
+	var pathToDesiredState=lifeCycleRobot.move(DEFAULT_LIFECYCLE,START_STATE,DESIRED_LIFECYCLE_STATE);
+
+	/*
+	 The function initializes an asset by checking if an assets rxt is present
+	 in the registry.If it is not present then rxt template is copied to the directory
+	 @context: An object containing a reference to the root of an asset
+	 */
+
+	function onAssetTypeInitialisation(context){
```

Comment:
Acknowledge, this code was belongs to IoT server ES extension app which is deprecated after moving to UUF based devicemgt app. So unable to fix the formatting issue.